src/clr-core/common/mixins/css-helpers.spec.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { html, LitElement } from 'lit-element';
+import { registerElementSafely } from '../utils/register';
+import { componentIsStable, createTestElement, removeTestElement, waitForComponent } from './../../test/utils';
+import { applyMixins } from './apply-mixins';
+import { CssHelpers } from './css-helpers';
+
+class CssHelpersTestBaseElement extends LitElement {}
+
+applyMixins(CssHelpersTestBaseElement, [CssHelpers]);
+
+class CssHelpersTestElement extends CssHelpersTestBaseElement {
+  render() {
+    return html`ohai`;
+  }
+}
+interface CssHelpersTestElement extends CssHelpersTestBaseElement, CssHelpers {}
+registerElementSafely('css-helpers-test-element', CssHelpersTestElement);
+
+describe('CssHelpers mixin - ', () => {
+  let testElement: HTMLElement;
+  let component: CssHelpersTestElement;
+  const testClassname = 'i-have-this-class';
+  const failClassname = 'jabberwocky';
+
+  beforeEach(async () => {
+    testElement = createTestElement();
+    testElement.innerHTML = `
+      <css-helpers-test-element class="${testClassname}">
+      </css-helpers-test-element>
+    `;
+    await waitForComponent('css-helpers-test-element');
+    component = testElement.querySelector<CssHelpersTestElement>('css-helpers-test-element');
+  });
+
+  afterEach(() => {
+    removeTestElement(testElement);
+  });
+
+  it('should have the helper methods on the component', async () => {
+    await componentIsStable(component);
+    expect(component.hasClassname).toBeTruthy();
+    expect(component.addClassname).toBeTruthy();
+    expect(component.removeClassname).toBeTruthy();
+    expect(component.removeClassnames).toBeTruthy();
+    expect(component.removeClassnamesUnless).toBeTruthy();
+    expect(component.updateEquilateralStyles).toBeTruthy();
+    expect(component.removeEquilateralStyles).toBeTruthy();
+    expect(component.addEquilateralStyles).toBeTruthy();
```

Comment:
Yeah. This test is only testing whether the mixin was applied. So one item passing infers that all items are going to pass. This would fail if we removed a method from CssHelpers. But not if we added one.